src/commands/app/init.js

```diff
@@ -135,32 +135,33 @@ class InitCommand extends BaseCommand {
   async selectExtensionPoints (flags, orgSupportedServices = null) {
     if (!flags.extensions) {
       return [implPromptChoices.find(i => i.value.name === 'application').value]
+    } else if (flags.extension) {
+      return implPromptChoices.filter(i => flags.extension.indexOf(i.value.name) > -1)
```

Comment:
needs a meaning full error if the list is empty meaning the specified extension is invalid